inspirehep/modules/orcid/static/templates/match_profile.html

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
This is a jinja2 template so should be in `inspirehep/modules/orcid/templates/orcid`

`static` is reserved for static files (JS, Angular templates and so on)